c/frame/py_frame_init.cc

```diff
@@ -118,10 +120,17 @@ class FrameInitializationManager {
       if (src.is_undefined() || src.is_none()) {
         return init_empty_frame();
       }
+      if (src.is_pandas_frame() || src.is_pandas_series()) {
+        return init_from_pandas();
+      }
+      if (src.is_numpy_array()) {
+        return init_from_numpy();
+      }
       if (src.is_ellipsis() &&
                !defined_names && !defined_stypes && !defined_stype) {
         return init_mystery_frame();
```

Comment:
What is the ellipsis and the mystery frame? :)

Reply:
`dt.Frame(...)`  🤫